### ISSUE TITLE
[Bug Report] Incorrect dof max velocities for explicit actuators

### ISSUE THREAD

**Issue by diegoaldarondo-fauna** [isaac-sim/IsaacLab#1384]
### Describe the bug

When using explicit actuators in an articulation, the max dof velocity is not set properly. For explicit actuators, the articulation root_physx_view seemingly should set the max_dof_velocity of the implicit actuator to a large number, similar to how effort limits are handled. This is particularly true for explicit actuators that implement a velocity limit, but exactly how it should be handled in general is unclear. 

### Steps to reproduce

Instantiate any model using explicit actuators and velocity limits greater than the default physx max dof velocity limits. 

### System Info

Describe the characteristic of your environment:

- Commit: b9a49ca
- Isaac Sim Versions: 4.0.0-4.2.0
- OS: [e.g. Ubuntu 22.04]
- GPU: [e.g. NVIDIA RTX 4090]
- CUDA: [e.g. 12.1]
- GPU Driver: [e.g. 535.183.01 and 550.120]

### Checklist

- [x] I have checked that there is no similar issue in the repo (**required**)
- [x] I have checked that the issue is not in running Isaac Sim itself and is related to the repo

### Acceptance Criteria

- [ ] The velocity limits should be lifted for explicit actuators.

**Comment by jtigue-bdai** [comment]
Currently for explicit actuators(i.e. `ActuatorBase`), the `velocity_limit` is only used by the `DCMotor` actuator. It is used to calculate the motor torque limit. Currently nothing is setting the velocity limits of the `root_physx_view`. If the articulation USD you are importing has implicit actuator limit set then those will be unchanged. 

Am I correct in my understanding that you would like the explicit actuators to overwrite any imported articulation joint limits via  root_physx_view.set_dof_max_velocities.

**Comment by diegoaldarondo-fauna** [comment]
Yes, I believe that is the case, at least for explicit actuators that define a `velocity_limit`. Alternatively, I could imagine a warning when the DCMotor velocity limit exceeds the imported articulation joint limit. 

I mainly am interested in avoiding a situation in which the `DCMotor` velocity limits are accidentally/silently rendered somewhat irrelevant due to the imported articulation velocity limit.

**Comment by jtigue-bdai** [comment]
Ok thanks for the clarification. I will add this to our list.

**Comment by Mayankm96** [comment]
We have been having follow-up discussions on the MR itself. Might make sense to discuss them here.

https://github.com/isaac-sim/IsaacLab/pull/1509#issuecomment-2532097391

**Comment by larsrpe** [comment]
I agree that it should be very clear which settings are propagated to the simulation-level. For now I think the only potential issue with this MR is for `DCMotor`, so to patch this separating the explicit model velocity limit (no load speed) and solver level velocity limits might be a good start?

**Comment by Mayankm96** [comment]
Is anyone looking into this? The recent change affects the way limits are set in the simulator and might affect sim-to-real for some applications where this is unintentional.

@jsmith-bdai @kellyguo11

**Comment by jtigue-bdai** [comment]
> Is anyone looking into this? The recent change affects the way limits are set in the simulator and might affect sim-to-real for some applications where this is unintentional.
> 
> @jsmith-bdai @kellyguo11

I have it on my list of To Dos, but I haven't gotten to it yet.
